discourse_rock/rockcraft.yaml

```diff
@@ -280,6 +279,108 @@ parts:
       chown -R 584792:584792 var/lib/pebble/default/.npm
       chown -R 584792:584792 var/lib/pebble/default/.yarn
       chown    584792:584792 var/lib/pebble/default/.yarnrc
+  imagemagick:
```

Comment:
Could you add a note in the PR description about why are you adding this imagemagick now please?

Reply:
Good point, thanks!